CANTroller2/src/RunModeManager.h

```diff
@@ -132,8 +132,7 @@ class RunModeManager {
             if (!car_stopped() && !stopcarTimer.expired()) pressure_target_psi = min (pressure_target_psi + pressure_hold_increment_psi, pressure_sensor.get_max_human());  // If the car is still moving, push harder
         }
         if (ctrl_pos_adc[VERT][FILT] < ctrl_db_adc[VERT][TOP]) joy_centered = true; // Mark joystick at or below center, now pushing up will go to fly mode
-        else if (joy_centered && (ctrl == JOY || hotrc_radio_detected)) updateMode(FLY); // Enter Fly Mode upon joystick movement from center to above center.
-        // Possibly add "&& car_stopped()" to above check?
+        else if (joy_centered && (ctrl == JOY || hotrc_radio_detected)) updateMode(FLY); // Enter Fly Mode upon joystick movement from center to above center  // Possibly add "&& car_stopped()" to above check?
```

Comment:
I'm unclear on a bunch of this. Do we also use a joystick? If we're not using a joystick, maybe we should update the code. So then we check if ctrl (and ctrl determines what device eg joystick, hotrc, sim, headless is controlling something... but what is it controlling? Just the runmode? Based on line 134 where we check ctrl_pos_adc[vert][filt] I'm assuming it's only using the vertical input from the joystick). 

So we're checking if joy_centered is true. Based on the name, joy_centered is a bool that's true if the joystick is centered, but I'm not sure what that means specifically.
If eg assuming joystick is now hotrc, it means that the vertical trigger is released / centered / in the dead zone, could we just have a check like hotrc_vert.is_centered()? This could check the latest read against a range. Based on the comment we're actually wanting to check if at or below center, so maybe a different name makes sense. It would also replace the line 134 code with something more readable. 

Anyway if joy_centered and we're using the joystick OR hotrc_radio_detected we should enter fly mode. Based on the variable names here, we're checking to see if the joystick is NOT being pulled but is active as the controller, or if the hotrc is turned on and connected - but again, nothing indicates that we've input a movement from center to above center. 

RE adding && car_stopped(), what scenarios exist where the car is in hold mode, and is still moving, and you would want to enter fly mode? I'm guessing it went into hold mode and was still moving, so it starts applying brakes, and you don't want it to stop so you pull on the trigger and enter fly mode. If you add this && car_stopped check, any time you enter hold mode you'll have to wait until a complete stop to get to fly mode. Am I understanding this right?